src/stories/TextInputWithTokens.stories.tsx

```diff
@@ -30,7 +32,53 @@ export default {
         </ThemeProvider>
       )
     }
-  ]
+  ],
+  argTypes: {
+    maxHeight: {
+      defaultValue: undefined,
+      control: {
+        type: 'text'
+      }
+    },
+    preventTokenWrapping: {
+      defaultValue: false,
+      control: {
+        type: 'boolean'
+      }
+    },
+    size: {
+      name: 'size (token size)',
+      defaultValue: 'extralarge',
+      options: ['small', 'medium', 'large', 'extralarge'],
+      control: {
+        type: 'radio'
+      }
+    },
+    inputSize: {
```

Comment:
Heads up.. this control doesn't seem to work as intended?

https://user-images.githubusercontent.com/13340707/152334520-8970f2da-9b7e-42b7-a59b-ff05bf8eba98.mov

Reply:
Not sure what happened there. Will fix.

Reply:
The size difference isn't noticeable with most token sizes unless you have text in the input.

Maybe we should get rid of the `inputSize` prop altogether, and rely just on the `size` prop. Then we can manually pass the size to the `TextInputWrapper` depending on what was passed to `size`.

I'll push something up and you can let me know what you think.

Reply:
Before:
https://user-images.githubusercontent.com/2313998/153096054-74a5d87e-646e-4a5c-84b0-539d87a428eb.mp4

After:
https://user-images.githubusercontent.com/2313998/153096410-3d8c7d3d-4f3a-4c2c-83b6-35ad8cc5dc38.mp4

Reply:
Yep, I think this is a big improvement 👍 . Thanks for making this change.

On a side note, are there usage guidelines for this particular prop? Curious why / where you'd need some of these larger sizes. Could we be encouraging an anti-pattern?

Reply:
I think we would mostly use the larger sizes because they have a larger touch target. You would use different sizes depending on where you're rendering the input.